src/app/react/v5/[...slug]/page.tsx

```diff
@@ -0,0 +1,29 @@
+import notFound from "../../../not-found";
+import { fetchTypeScriptDoc } from "../../../references/components/TDoc/fetchDocs/fetchTypeScriptDoc";
+import { RootTDoc } from "../../../references/components/TDoc/Root";
+import { getSlugToDocMap } from "../../../references/components/TDoc/utils/slugs";
+
+type PageProps = { params: { slug: string[] } };
+
+export default async function Page(props: PageProps) {
+	const doc = await fetchTypeScriptDoc("v5");
+	const slugToDoc = getSlugToDocMap(doc);
+	const docSlug = props.params.slug?.join("/");
+
+	if (!docSlug) {
+		notFound();
+	}
+
+	// API page
+	const selectedDoc = docSlug && slugToDoc[docSlug];
+
+	if (selectedDoc) {
+		return (
+			<div>
+				<RootTDoc doc={selectedDoc} />
```

Comment:
This makes the pages for all the APIs in /react that do not belong in react

Reply:
same as above - what's the downside?

Reply:
That's it, /react/v5/ has the ability to create pages for non-react stuff in that path - which seems weird. Also, these are generated on-demand and not at build time